### PR TITLE
Multihost, yo

### PR DESCRIPTION
**Pull request by mdarby**
@carolynvs what say you?

You'll need to add `export DOCKER_HOSTS='oauth oauth2'` to your `vars` file.

### CODE REVIEW

File: bin/lib.sh [hunk 1]
```diff
@@ -63,16 +63,27 @@ check_env() {
...
 
-  deploy_app
+    deploy_app
+  done
+
+  clear
+
+  for i in $DOCKER_HOSTS; do
+    eval "$(docker-machine env $i)"
+    docker ps
```
Comment (carolynvs) [inline bin/lib.sh]: What is this part for?

Comment (mdarby) [inline bin/lib.sh]: It clears the screen and loops through each host and displays the containers.

Comment (carolynvs) [inline bin/lib.sh]: Doh! Can't believe I missed that, I need a nap. :smile:

**Comment by carolynvs** [comment]
:shipit: :rocket:

**Pull request merged**